src/routes/FarmHome.vue

```diff
@@ -43,89 +65,72 @@
 
 
 <script>
-  let farmid
+  import * as api from '../api/general'
+  import settings from '../config/settings'
+  let farm_id, farm, all_products
   export default {
     /* global localStorage:true */
     data () {
+      // get user info
       this.$http.get('http://ec2-52-56-114-123.eu-west-2.compute.amazonaws.com/auth/user', {
```

Comment:
Replaced them. If I missed some link, don't hesitate to call my attention to it :) .